src/tests/test_buildercore_cfngen.py

```diff
@@ -35,3 +35,9 @@ def test_validation_elife_projects(self):
             sleep(0.25)
 
         self.switch_in_test_settings()
+
+    def test_empty_template_delta(self):
```

Comment:
Would like to, I'll have to manually modify the template, because rendering will always produce the same result.